Duplicati/Server/WebServer/RESTMethods/RemoteOperation.cs

```diff
@@ -66,7 +65,7 @@ private void ListFolder(string uri, RequestInfo info)
 
         private void TestConnection(string url, RequestInfo info)
         {
-            
+            bool autoCreate = false;

```

Comment:
As said below, I would use the query string of the original request:
```cs
bool autoCreate = info.Request.QueryString["autocreate"] == "true";
```
or something similar